zabbix_metrics_watcher.py

```diff
@@ -52,47 +51,53 @@ def is_finished(self):
     def duration(self):
         try:
             return int(self._data['status']['duration'])/1000000000
-        except:
+        except Exception as e:
+            logger.warn('Error duration: %s' % repr(e))
```

Comment:
Can use `%r` and also avoid immediate %-subst by just supplying `e` as an extra argument.